Packaging/resources/assets/txtdata/Experience.txt

```diff
@@ -0,0 +1,53 @@
+Level	Experience
+MaxLevel	50

```

Comment:
Let's just use the last row as MaxLevel?

Reply:
This was only included to match the format expected by Diablo 2 textfiles. Removing the parsing for that row is easy enough though (skipping it if it's present in the file) if we're willing to accept the potential confusion for people used to D2 modding?

Reply:
Ah, didn't realize D2 used the same format

Reply:
I think it's ok to not be 1:1 with D2 data tables.

For one thing I never looked at them :D

I also prefer .tsv even if it requires people that use D2 specific tools to have to rename or force open files. I would rather leave a line on that in tutorial somewhere.